app-crypt/argon2/argon2-20171227.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="A password-hashing function"
+DESCRIPTION="Argon2 is a password-hashing function that summarizes the state \
+of the art in the design of memory-hard functions and can be used to hash \
+passwords for credential storage, key derivation, or other applications."
+HOMEPAGE="https://github.com/p-h-c/phc-winner-argon2"
+COPYRIGHT="2015-2017 Daniel Dinu
+	2013-2017 Samuel Neves
+	2015-2017 Dmitry Khoravich
+	2015-2017 Jean-Philippe Aumasson
+	2015 Thomas Pornin"
+LICENSE="Apache v2
+	Public Domain"
+REVISION="1"
+SOURCE_URI="https://github.com/P-H-C/phc-winner-argon2/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="eaea0172c1f4ee4550d1b6c9ce01aab8d1ab66b4207776aa67991eb5872fdcd8"
+SOURCE_FILENAME="argon2-$portVersion.tar.gz"
+SOURCE_DIR="phc-winner-argon2-$portVersion"
+PATCHES="argon2-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	argon2$secondaryArchSuffix = $portVersion
+	cmd:argon2$secondaryArchSuffix = $portVersion
```

Comment:
Maybe we could skip the cmd here for _x86?

Reply:
Sure.